pkgs/development/libraries/pipewire/default.nix

```diff
@@ -172,6 +173,11 @@ let
       patchShebangs source/doc/input-filter.sh
       patchShebangs source/doc/input-filter-h.sh
       patchShebangs source/spa/tests/gen-cpp-test.py
+
+      # Don't try to put system service units into pkgs.systemd.
+      sed -Ei "
+        s@^(systemd_system_services_dir).*@\\1 = '$out/lib/systemd/system'@
```

Comment:
Should be fixed by https://gitlab.freedesktop.org/pipewire/pipewire/-/commit/5054b48c9de655b4b48f7c801cb305d9eb122520

Reply:
Thanks for the pointer, I'll check that.

Reply:
Looks like this patch hasn't been released yet, so I'm gonna reference it in `patches`.